src/python/pants/backend/terraform/dependencies.py

```diff
@@ -107,24 +106,22 @@ class TerraformInitRequest:
 
     # Not initialising the backend means we won't access remote state. Useful for `validate`
     initialise_backend: bool = False
+    upgrade: bool = False
 
 
 @dataclass(frozen=True)
-class TerraformInitResponse:
-    sources_and_deps: Digest
-    terraform_files: SourceFiles
-    chdir: str
-
+class TerraformThingsNeededToRun:
```

Comment:
Not mad about this name TBH, although it is very descriptive. `TerraformCommandNeeds`? `TerraformCommandRequirements`?

Reply:
hecc I thought I replaced all the silly names